### ISSUE TITLE
Add HtmlMarker.Popup

### ISSUE THREAD

**Issue by IVData** [arnaudleclerc/AzureMapsControl.Components#33]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to easily add a popup to an HtmlMarker

**Describe the solution you'd like**
Add HtmlMarker.Popup and HtmlMarker.TogglePopup() member as in the top example on this page: https://docs.microsoft.com/en-us/azure/azure-maps/map-add-custom-html

**Describe alternatives you've considered**
I think the alternative here is to define the popup separately, probably store all HtmlMarkers and Popups in separate dictionaries so when a marker is clicked on you can find the associated popup. Which just seems a bit messier.

**Comment by arnaudleclerc** [comment]
That's a good idea, thanks for the suggestion! I have to check exactly how a popup attached to an HTML Marker behaves during the clear operations and if it is accessible without the HTML Marker. I will try to take a look at it in the next couple of days

**Comment by IVData** [comment]
Awesome, thanks!

BTW, this repo is the only good looking Bing Maps for blazor repo I could find, and it's amazing. Thank you!

**Comment by arnaudleclerc** [comment]
There is still a couple of things to do, mainly because the Popup should also be usable outside of the HtmlMarker`s context, but here is a sneak peak of how the example you provided would look like : 

```
@page "/HtmlMarkerPopup"

@using AzureMapsControl.Components.Map
<AzureMap Id="map"
          EventActivationFlags="MapEventActivationFlags
                                .None()
                                .Enable(MapEventType.Ready)"
          OnReady="OnMapReady" />

@code  {
    public async Task OnMapReady(MapEventArgs events)
    {
        var marker = new AzureMapsControl.Components.Markers.HtmlMarker(
            new Components.Markers.HtmlMarkerOptions
            {
                Position = new Components.Atlas.Position(0, 0),
                Draggable = true,
                Popup = new Components.Popups.HtmlMarkerPopup(new Components.Popups.PopupOptions
                {
                    Content = "<di style='padding:10px'v>Hello World</div>",
                    PixelOffset = new Components.Atlas.Pixel(0, -30)
                })
            })
        {
            EventActivationFlags = Components.Markers.HtmlMarkerEventActivationFlags.None().Enable(Components.Markers.HtmlMarkerEventType.Click)
        };

        await events.Map.AddHtmlMarkersAsync(marker);

        marker.OnClick += async (eventArgs) =>
        {
            await marker.TogglePopupAsync();
        };
    }
}
```

**Comment by arnaudleclerc** [comment]
@IVData  I have published a version 0.16.0 with this new functionality. Let me know if that fits your needs. I will let you close the issue if everything is fine for you.

**Comment by IVData** [comment]
@arnaudleclerc  this is great! Thanks for such a fast response. I've tested it out and it works as expected.

Unfortunately it looks like I have too many markers and might have to use a symbol layer and make my own popups to increase performance =\ There are examples of doing this from MS too though.

Thanks again
